waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/metrics/MonitoredAspect.java

```diff
@@ -70,24 +71,20 @@ public Object monitor(ProceedingJoinPoint pjp, Monitored monitored) throws Throw
   }
 
   @VisibleForTesting
-  void setCounterService(CounterService counterService) {
-    this.counterService = counterService;
-  }
-
-  @VisibleForTesting
-  void setGaugeService(GaugeService gaugeService) {
-    this.gaugeService = gaugeService;
+  void setMeterRegistry(GraphiteMeterRegistry graphiteMeterRegistry) {
```

Comment:
Can the type of the paramater not be `MeterRegistry`? Ideally we don't want to couple this class with Graphite.

Reply:
Absolutely. I missed that during code cleanup yesterday. Have changed it now.